src/index.html

```diff
@@ -11,7 +11,127 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="label">Surname:
+          <input 
+            type="text" 
+            name="surname" 
+            autocomplete="off"
+            required="required"
```

Comment:
if the value of attribute is equal to the attribute name, you can skip the setting value
```suggestion
            required
```